### DISCUSSION TITLE
Taskbar Glitches During Remote Desktop (RDP) session (using W11 Home Connect to W10 Pro)

### DISCUSSION THREAD

**Discussion by SustainLessString** [valinet/ExplorerPatcher#726] [Q&A]
The glitches happened often when using W10 Taskbar Style and some times, rarely, with W11 Taskbar Style. The ExplorerPatcher version used is 2200.376.40.21.
Client Host is on Windows 11 Home Single Language version 21H2 Build 2200.438 with currently active notification tray by default windows 11 +(Time and Date, Battery Indicator, Volume Indicator and WiFi Indicator) + Language Input switches.
Remote Hos is on Windows 10 Pro version 21H2 Build 19044.1469 with currently active notification tray:
Kaspersky End Point Security, Telegram Desktop, Network Indicator, Volume Indicator.
The bottom right taskbar on my Client usually fuses with Remote Client in very disturbing way. They usual action I take is just to hide all off them but the hiding the Telegram Desktop during work hours puts me in a tight spot since I could not get notified whenever new message pops up.

**Comment by tomshaffner** [comment]
I'm having the same issue; created a bug report for it at https://github.com/valinet/ExplorerPatcher/issues/4174.
